lib/iiif_print/split_pdfs/adventist_pages_to_jpgs_splitter.rb

```diff
@@ -16,13 +16,14 @@ module AdventistPagesToJpgsSplitter
       # @note I am adding a {.new} method to a module to mimic the instantiation of a class.
       #
       # @see https://github.com/scientist-softserv/iiif_print/blob/a23706453f23e0f54c9d50bbf0ddf9311d82a0b9/lib/iiif_print/jobs/child_works_from_pdf_job.rb#L39-L63
-      # rubocop:disable Metrics/LineLength
-      def self.new(path, *args, splitter: PagesToJpgsSplitter, suffixes: CreateDerivativesJobDecorator::NON_ARCHIVAL_PDF_SUFFIXES)
+      def self.call(path,
+                    splitter: PagesToJpgsSplitter,
+                    suffixes: CreateDerivativesJobDecorator::NON_ARCHIVAL_PDF_SUFFIXES,
+                    **args)
         return [] if suffixes.any? { |suffix| path.downcase.end_with?(suffix) }
 
-        splitter.new(path, *args)
+        splitter.call(path, args)
```

Comment:
I think we want to preserve the keyword nature of the `args`:

```suggestion
        splitter.call(path, **args)
```

Looking at [IiifPrint::SplitPdfs::BaseSplitter.call](https://github.com/scientist-softserv/iiif_print/blob/bdadcc04d971a9e8bb08ffd79b5bf012a072b34a/lib/iiif_print/split_pdfs/base_splitter.rb#L24-L26) it completely swallows those args.